backoffice/src/views/exports/certificates/CertificateView.tsx

```diff
@@ -34,20 +34,35 @@ export const CertificateView: FunctionComponent<CertificateViewProps> = ({
   }
 };
 
-export const LetterView: FunctionComponent<CertificateViewProps> = ({
+interface LetterViewProps {
+  exportsGateway: IExportsGateway;
+  beaconId: string;
+  letterType: string;
+}
+
+export const LetterView: FunctionComponent<LetterViewProps> = ({
   exportsGateway,
   beaconId,
+  letterType,
 }): JSX.Element => {
   const [beacon, setBeacon] = useState<IBeaconExport>({} as IBeaconExport);
+  console.log(letterType);
 
   useEffect(() => {
     exportsGateway.getExportDataForBeacon(beaconId).then(setBeacon);
   }, [beaconId, exportsGateway]);
 
-  return (
-    <div>
-      <CoverLetter beacon={beacon} type="Registration" />
-      <CoverLetter beacon={beacon} type="Amended" />
-    </div>
-  );
+  if (letterType === "registration") {
```

Comment:
How would you feel about this sort of syntax instead?

return(
<div>
{letterType === "registration" && <CoverLetter beacon={beacon} type="Registration" />}
{letterType === "amended" && <CoverLetter beacon={beacon} type="Registration" />}
</div>
}